src/ceph.in

```diff
@@ -523,7 +523,7 @@ def ceph_conf(parsed_args, field, name):
         stdout=subprocess.PIPE,
         stderr=subprocess.PIPE)
     outdata, errdata = p.communicate()
-    if len(errdata):
+    if len(errdata) and p.returncode != 0:
```

Comment:
see https://github.com/ceph/ceph/blob/master/doc/rados/configuration/ceph-conf.rst#changes-since-nautilus 

probably we should test this change on master and mark the tracker ticket so it can be backported properly. and use 
```suggestion
    if p.returncode != 0:
```
instead, if we want to be more tolerant. as i think the returncode is the only authentic indicator to tell if a command is successful or not.